app/directives/my-pipeline-configurations/my-batch-pipeline-config/my-batch-pipeline-config.html

```diff
@@ -206,11 +206,11 @@
           store="BatchPipelineConfigCtrl.store">
         </my-post-run-actions>
       </div>
-      <div class="configuration-step-content" id="pushdown-tab-content"
-        ng-if="BatchPipelineConfigCtrl.activeTab === 'pushdown'" style="height: 100%; width: 100%; display: block;">
+      <div class="configuration-step-content pushdown-config-full-block" id="pushdown-tab-content"
+        ng-if="BatchPipelineConfigCtrl.activeTab === 'pushdown'">
         <pushdown-config value="BatchPipelineConfigCtrl.pushdownConfig"
           on-value-change="BatchPipelineConfigCtrl.onPushdownConfigChange"
-          style="height: 100%; width: 100%; display: block;">
+          class="pushdown-config-full-block">
```

Comment:
just verifiying, is this class required at both levels?

Reply:
It is. It's to inflate/limit them to the full width and height of the parent, so that the styling in the react component can respond to that height to do scrolling without simply overflowing the configuration popup. An alternative could be to expand the entire configuration popup dynamically - it didn't happen naturally, and I didn't try to scope it, might not be too bad. This way was an easier path and I was being a greedy planning algorithm at the time.